openvpn-install.sh

```diff
@@ -670,6 +670,7 @@ function installOpenVPN () {
 
 	# Workaround to remove unharmful error until easy-rsa 3.0.7
 	# https://github.com/OpenVPN/easy-rsa/issues/261
+	cd /etc/openvpn/easy-rsa/
```

Comment:
There is already the same `cd` at line 655 🤔

Reply:
yeah,  because centos 8 gave error about:
```
sed: can't read pki/openssl-easyrsa.cnf: No such file or directory
```
The problem source this line : **sed -i 's/^RANDFILE/#RANDFILE/g' pki/openssl-easyrsa.cnf** line. I had add this line but I think is not necessary.

I had debug with **-f filename**:
![angristan2](https://user-images.githubusercontent.com/43282745/67435666-9c820680-f5ba-11e9-96fd-189bcf7fb783.png)


It's reason, didn't exist openssl-easyrsa.cnf file this step so it didn't find find file. The file created after **./easyrsa init-pki** line. 

I had try this codes and worked:
```
        # Create the PKI, set up the CA, the DH params and the server certificate
        ./easyrsa init-pki

        # Workaround to remove unharmful error until easy-rsa 3.0.7
        # https://github.com/OpenVPN/easy-rsa/issues/261
         sed -i 's/^RANDFILE/#RANDFILE/g' pki/openssl-easyrsa.cnf

         ./easyrsa --batch build-ca nopass
```
Is that way fine your ?

Reply:
I changed line like comment up there. @angristan

Reply:
also i had forgot write my system properties:

LSB Version:	:core-4.1-amd64:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch